docs/djavalangstringsubstringnocopy.md

```diff
@@ -0,0 +1,41 @@
+<!--
```

Comment:
This topic needs adding to the `nav` section of the `mkdocs.yml` file, in alphabetical order under "System property options". The entry will look something like this, but should be formatted and indented to be consistent:
`- "-Djava.lang.string.substring.nocopy" : djavalangstringsubstringnocopy.md`
(Links to the topic work OK because MkDocs compiles any file it finds, but unless it's listed in the `.yml` file it's not indexed nor in the table of contents!)

Reply:
Done